bootstrap/templates/terraform/tf-state-bucket.tf.in

```diff
@@ -13,22 +13,22 @@
 # limitations under the License.
 
 module "tf-state-bucket" {
-  source     = "github.com/metro-digital/terraform-google-cf-bucket.git?ref=latest"
+  source = "github.com/metro-digital/terraform-google-cf-bucket.git?ref=latest"
```

Comment:
We should use the module registry part here: `source  = "metro-digital/cf-bucket/google"`